### PR TITLE
reference: update Tispark FAQ

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR updates TiSpark FAQ in `tispark.md`.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1900

### Which version does your change affect? 
dev, v2.1, v3.0

### CODE REVIEW

File: dev/reference/tispark.md [hunk 1]
```diff
@@ -262,3 +262,7 @@ A: You can ignore this warning. It occurs because Spark tries to load two nonexi
 Q: What can I do if `java.sql.BatchUpdateException: Data Truncated` is returned when executing SQL statements using TiSpark?
 
 A: This error occurs because the length of the data written exceeds the length of the data type defined by the database. You can check the field length and adjust it accordingly.
+
+Q: Does TiSpark task read Hive's metadata by default?
+
+A: TiSpark searches for the hive library by reading the meta in the hive-site. If TiSpark fails to find it, TiSpark searches for the tidb library instead by reading tidb meta. Do not configure hive's meta in hive site if it is unnecessary for you.
```
Comment (TomShawn) [inline dev/reference/tispark.md]:
```suggestion
A: TiSpark searches for the Hive database by reading the Hive metadata in hive-site. If the search task fails, it searches for the TiDB database instead, by reading the TiDB metadata. 

To disable this default behavior, do not configure the Hive metadata in hive-site.
```

**Comment by anotherrachel** [comment]
@TomShawn All comments addressed. PTAL again, thanks~

**Comment by TomShawn** [comment]
LGTM
@King-Dylan PTAL, thanks.

File: v2.1/reference/tispark.md [hunk 2]
```diff
@@ -271,3 +271,9 @@ A: You can ignore this warning. It occurs because Spark tries to load two nonexi
 Q: What can I do if `java.sql.BatchUpdateException: Data Truncated` is returned when executing SQL statements using TiSpark?
 
 A: This error occurs because the length of the data written exceeds the length of the data type defined by the database. You can check the field length and adjust it accordingly.
+
+Q: Does TiSpark read Hive metadata by default?
+
+A: TiSpark searches for the Hive database by reading the Hive metadata in hive-site. If the search task fails, it searches for the TiDB database instead, by reading the TiDB metadata.
+
+To disable this default behavior, do not configure the Hive metadata in hive-site.
```
Comment (lilin90) [inline v2.1/reference/tispark.md]: This sentence is a little bit weird to me. How about:
```suggestion
If you do not need this default behavior, do not configure the Hive metadata in hive-site.
```

File: v2.1/reference/tispark.md [hunk 3]
```diff
@@ -271,3 +271,9 @@ A: You can ignore this warning. It occurs because Spark tries to load two nonexi
 Q: What can I do if `java.sql.BatchUpdateException: Data Truncated` is returned when executing SQL statements using TiSpark?
 
 A: This error occurs because the length of the data written exceeds the length of the data type defined by the database. You can check the field length and adjust it accordingly.
+
+Q: Does TiSpark read Hive metadata by default?
+
+A: TiSpark searches for the Hive database by reading the Hive metadata in hive-site. If the search task fails, it searches for the TiDB database instead, by reading the TiDB metadata.
```
Comment (lilin90) [inline v2.1/reference/tispark.md]:
```suggestion
A: TiSpark searches for the Hive database by reading the Hive metadata in hive-site by default. If the search task fails, it searches for the TiDB database instead, by reading the TiDB metadata.
```

**Comment by lilin90** [comment]
@King-Dylan PTAL

**Pull request merged**